yt/frontends/enzo_p/data_structures.py

```diff
@@ -278,6 +317,10 @@ def __init__(self, filename, dataset_type=None,
         self.storage_filename = storage_filename
         Dataset.__init__(self, filename, dataset_type, file_style=file_style,
                          units_override=units_override, unit_system=unit_system)
+        mylog.warning(
+            "The Enzo-P file format is still under development and may " +
+            "change. If loading fails, simulation data will need to be " +
+            "re-generated.")
```

Comment:
Can you make this use `UserWarning` or a subclass of `UserWarning`? Not everyone has logging enabled and I think this message is worth being noisier than usual about.

Reply:
Sure thing.